array-manipulation-1/src/head.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-unused-vars */
+
+function head(array) {
+  if (array.length > 0) {
+    return array[0];
+  }
+  return undefined;
+}
```

Comment:
If the array length was not greater than zero, it would return undefined anyway.